pkg/operator/apiservercontrollerset/dsnodeprovider.go

```diff
@@ -7,12 +7,11 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	encryptiondeployer "github.com/openshift/library-go/pkg/operator/encryption/deployer"
-
-	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/operatorclient"
 )
 
 // DaemonSetNodeProvider returns the node list from nodes matching the node selector of a DaemonSet
 type DaemonSetNodeProvider struct {
+	TargetNamespace                  string
```

Comment:
this one is for encryption?  move to an encryption package, not an apiserver one for now.